File: C#/Object-Oriented-Programming/Encapsulation/Encapsulation.cs
using Object_Oriented_Programming.EncapsulationImpl;

namespace Object_Oriented_Programming
{
    class Encapsulation
    {
        public static void Invoke()
        {
            Console.WriteLine("Encapsulation");
            Employee employee = new Employee();
            employee.EmployeeId = 2;
            employee.EmployeeName = "Jane Doe";
            employee.Display();
        }

    }
}

File: Java/Object-Oriented-Programming/Encapsulation/Encapsulation.java
package ObjectOrientedProgramming.Encapsulation;

public class Encapsulation {
    public static void invoke() {
        System.out.println("Encapsulation");
        Employee employee = new Employee();
        employee.setId(2);
        employee.setName("Jane Doe");
        employee.display();
    }
}